src/main.rs

```diff
@@ -66,16 +75,24 @@ enum Cli {
     /// Pay a number of bones to a given address. Note that 1 HNT is
     /// 100_000_000 bones
     Pay {
+        /// Whether to use Ledger
+        #[structopt(short = "l", long = "ledger")]
+        ledger: bool,
+
+        /// Whether to parse amount as bones
+        #[structopt(short = "b", long = "bones")]
```

Comment:
This feels dangerous to me.

Reply:
Maybe it is. I've taken steps to try to mitigate though, like a warning message on the transaction.